### PR TITLE
add sudo call to set tempo.

### PR DESCRIPTION
**Pull request by SaMotlagh**

### CODE REVIEW

**Review by S0AndS0 — APPROVED** [review]

File: pallets/subtensor/src/lib.rs [hunk 1]
```diff
@@ -1555,7 +1556,15 @@ pub mod pallet {
 		pub fn benchmark_drain_emission( _:OriginFor<T> ) -> DispatchResult {
 			Self::drain_emission( 11 );
 			Ok(())
-		} 
+		}
+    
+    #[pallet::call_index(50)]
```
Comment (S0AndS0) [inline pallets/subtensor/src/lib.rs]: Mixing tabs and spaces?

**Review by S0AndS0 — APPROVED** [review]

File: pallets/subtensor/src/lib.rs (hunk 1 above)
Comment (SaMotlagh) [reply pallets/subtensor/src/lib.rs]: I usually use tabs.

Comment (S0AndS0) [reply pallets/subtensor/src/lib.rs]: Same, and if I remember correctly tabs are more accessible too, but totally not critical and just something I couldn't help notice

> Note; once everyone is rebased with recent `origin/main` updates such things _should_ be less of oddity due to `.cargo-husky/hooks/post-commit` incrementally enforcing `.rustfmt.toml` configurations

**Review by camfairchild — APPROVED** [review]
LGTM

**Review by Rubberbandits — APPROVED** [review]

**Pull request merged**